docs/platform/porting/system.md

```diff
@@ -2540,3 +2540,356 @@ Example:
+
+#### Kernel module
+
+**stability-monitor** relies on kernel module, which collects information about processes state in the system in most resource efficient way. This raw data is exposed via `/proc/tsm` interface.
+
+The following resource statistics are collected:
+-  CPU - system time + user time [cpu_usage = d(system_time + user_time) / dt]
+-  Memory - RSS
+-  I/O - bytes read + bytes written
+-  FDs - number of open file descriptors
+
+#### Service
+
+Purpose of **stability-monitor** service is to:
+-  Periodically monitor process state (based on raw data provided by kernel module)
+-  Gather additional process specific data from running system (eg. app type & state from AUL and pkgmgr)
+-  Read and apply provided configuration to the system (apply configuration defined limits)
+-  Generate D-Bus signal (and abnormality report, if configured)
+
+Service monitors parameters as provided by "data source" modules (which are processing raw data provided by kernel module).
+
+#### Configuration
+
+The configuration uses a hierarchical layout which consists of a set of JSON files. The primary file is `/usr/lib/stability-monitor/default.conf`. Additional files can be placed in `/etc/stability-monitor.d` and are read in alphabetical order (such that `Z.conf` will override `A.conf`). Configuration files are merged together (not replaced).
+
+In the resulting JSON object, the top-level entries have keys corresponding to programs, for example:
+-  resourced
+-  org.tizen.calendar
+
+Program name should be application ID for applications and executable basename (from `/proc/<pid>/exe`) for other system processes.
+
+If a program does not have specific configuration, the appropriate class-wide configuration is used. The available classes are:
+-  **global**  
+     Global section for all processes
+
+-  **foreground_native_app**  
+     Native applications running in foreground
+
+-  **background_native_app**  
+     Native applications running in background
+
+-  **foreground_web_app**  
+     Web applications running in foreground
+
+-  **background_web_app**  
+     Web applications running in background
+
+-  **foreground_csharp_app**  
+     .NET applications running in foreground
+
+-  **background_csharp_app**  
+     .NET applications running in background
+
+-  **foreground_ui_widget**  
+     Widget applications running in foreground
+
+-  **background_ui_widget**  
+     Widget applications running in background
+
+-  **native_service**  
+     Service applications
+
+-  **other_service**  
+     Other system processes
```

Comment:
To maintain consistency within the page, 
Please add period after text in lines 2580, 2583, 2586, 2589, 2592, 2595, 2598 2601, 2604, 2607, 2610.